.github/workflows/pathogen-repo-ci.yaml

```
@@ -63,6 +63,14 @@ on:
           - conda
         required: false
 
+      workflow-root:
+        description: >-
+          Path to the root of the Snakemake workflow filder to run CI on.
```

```suggestion
          Path to the root of the workflow folder (e.g. one that contains a Snakefile) to run CI on.
```

1. Fix a typo
2. Use Snakemake as an example, not the assumed workflow engine